src/artemis/fast_grid_scan_plan.py

```diff
@@ -0,0 +1,109 @@
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
+
+from src.artemis.devices.eiger import DetectorParams, EigerDetector
+from src.artemis.devices.fast_grid_scan import (
+    FastGridScan,
+    GridScanParams,
+    set_fast_grid_scan_params,
+)
+import bluesky.preprocessors as bpp
+import bluesky.plan_stubs as bps
+from bluesky import RunEngine
+from bluesky.callbacks import LiveTable
+from bluesky.utils import ProgressBarManager
+
+from src.artemis.devices.zebra import Zebra
+from src.artemis.devices.det_dim_constants import EIGER2_X_16M_SIZE
+import argparse
+from epics import caput
+from src.artemis.devices.det_dist_to_beam_converter import (
+    DetectorDistanceToBeamXYConverter,
+)
+
+from ophyd.log import config_ophyd_logging
+from bluesky.log import config_bluesky_logging
+
+from bluesky.plans import scan
+from ophyd.sim import motor1
+
+config_bluesky_logging(file="/tmp/bluesky.log", level="DEBUG")
+config_ophyd_logging(file="/tmp/ophyd.log", level="DEBUG")
+
+# Clear odin errors and check initialised
+# If in error clean up
+# Setup beamline
+# Store in ISPyB
+# Start nxgen
+# Start analysis run collection
+
+DETECTOR = EIGER2_X_16M_SIZE
+USE_ROI = False
+GRID_SCAN_PARAMS = GridScanParams(
+    x_steps=5,
+    y_steps=10,
+    x_step_size=0.1,
+    y_step_size=0.1,
+    dwell_time=0.2,
+    x_start=0.0,
+    y1_start=0.0,
+    z1_start=0.0,
+)
+DETECTOR_PARAMS = DetectorParams(
+    current_energy=100,
+    exposure_time=0.1,
+    acquisition_id="test",
+    directory="/tmp",
+    prefix="file.h5",
```

Comment:
If, as I assume, these are intended as example parameters the extension should be removed, as it's incorrect and is added on by odin anyway

Reply:
Agreed, fixed